src/cogent3/phylo/tree_distance.py

```diff
@@ -41,16 +101,97 @@ def lin_rajan_moret(tree1: "PhyloNode", tree2: "PhyloNode") -> float:
 
     matching_distance = _matched_distance(vector1, vector2)
 
-    return float(matching_distance)
+    return matching_distance
+
+
+def rooted_robinson_foulds(tree1: "TreeNode", tree2: "TreeNode") -> int:
+    """calculate the robinson-foulds distance between two rooted trees.
+
+    for rooted trees, the robinson-foulds distance is defined as the
```

Comment:
This text should be under a Notes section and given proper English grammar formatting (sentences start with capital letter, etc..)